### PR TITLE
Assignment1

### PR DESCRIPTION
**Pull request by alexvthornton**

### CODE REVIEW

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 1]
```diff
@@ -84,13 +94,33 @@ public static Question[] LoadQuestions(string filePath)
 
                 Question question = new Question();
                 question.Text = questionText;
-                question.Answers = new string[3];
+
```
Comment (worseTyler) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:97]: When I forked your repo to test your code, the lack of this line of code made it so you're program wouldn't run and one of the tests wouldn't pass. When I added this line back all of your code ran and your tests passed

**Comment by worseTyler** [comment]
Assignment
Pull Request title includes "Assignment 1" in the title ✔
Pull Request targets Assignment 1 branch ✔
Issue 1: Application no longer crashes ❌
Issue 2: Unit test properly passes ✔
Extra Credit
Issue 3: Feature request implemented ✔
Issue 3: Feature request unit tested ✔

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs (hunk 1 above)
Comment (alexvthornton) [reply PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:98]: Thanks for catching that I don't know how that got deleted.

**Comment by breaman** [comment]
Overall not too bad. Had a couple of coding standards that needed to be applied, though.

**Pull request closed without merging**